django_project/certification/models/course.py

```diff
@@ -38,7 +38,15 @@ class Course(models.Model):
         max_length=200,
         default=_('English'),
         null=True,
-        blank=True,
+        blank=True
+    )
+
+    trained_competence = models.CharField(
+        help_text=_(
+            'Trained competence e.g. Plugin development.'),
+            max_length=200,
+            null=True,
+            blank=True
```

Comment:
__[Flake8]__

[E131] continuation line unaligned for hanging indent


<!-- Comment by [SideCI](https://sideci.com) -->